src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -50,13 +50,6 @@
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
   </ItemGroup>
 
-  <ItemGroup Label="VSSDK Test framework">
-    <!-- see https://github.com/SonarSource/sonarlint-visualstudio/issues/2760 -->
-    <!--<PackageReference Condition="$(VsTargetVersion) != '2022'" Include="Microsoft.VisualStudio.Sdk.TestFramework" Version="16.5.22-beta" />-->
-
-    <PackageReference Condition="$(VsTargetVersion) == '2022'" Include="Microsoft.VisualStudio.Sdk.TestFramework" Version="17.2.7" />
-  </ItemGroup>
-
```

Comment:
This package was only used by tests that are currently ignored because they don't work.
There are separate tickets for the failing tests - #2760.